src/rubrix/metrics/token_classification/metrics.py

```diff
@@ -54,16 +60,17 @@ def tokens_length(
 
 def _check_compute_for(compute_for) -> str:
     try:
+        compute_for = compute_for.lower().strip()
         return _ACCEPTED_COMPUTE_FOR_VALUES[compute_for]
     except KeyError:
-        raise f"Wrong compute_for value {compute_for}"
+        raise RuntimeError(f"Wrong value [{compute_for}] for compute_for arg")
```

Comment:
Instead of this function, I would copy the `ExplicitEnum` [by HF](https://github.com/huggingface/transformers/blob/558f8543ba3860c736a7a9a4176ac20f23f9d5a0/src/transformers/file_utils.py#L2063). Then you get this check and a nice error message for free. Then in the methods below you would need to instantiate a `ComputeFor` object, if the input is a string. Let's discuss this via slack maybe.

Reply:
Done! I've include `_missing_` method just in `ComputeFor` enum class. We can discuss and abord in a separate PR this mechanism for all defined enums in rubrix